genai-quickstart-pocs-python/amazon-bedrock-intelligent-document-processing-poc/README.md

```diff
@@ -0,0 +1,95 @@
+
+## Step 2:
+
+Set up a python virtual environment in the root directory of the repository and ensure that you are using Python 3.9. This can be done by running the following commands:
+
+```
+pip install virtualenv
+python3.10 -m venv venv
+```
+
+The virtual environment will be extremely useful when you begin installing the requirements. If you need more clarification on the creation of the virtual environment please refer to this [blog](https://www.freecodecamp.org/news/how-to-setup-virtual-environments-in-python/).
+After the virtual environment is created, ensure that it is activated, following the activation steps of the virtual environment tool you are using. Likely:
+
+```
+cd venv
+cd bin
+source activate
+cd ../../
+```
+
+After your virtual environment has been created and activated, you can install all the requirements found in the requirements.txt file by running this command in the root of this repos directory in your terminal:
+
+```
+pip install -r requirements.txt
+```
+
+## Step 3:
+
+Now that the requirements have been successfully installed in your virtual environment we can begin configuring environment variables. You will first need to create a .env file in the root of this repo. Within the .env file you just created you will need to configure the .env to contain your AWS profile, along with the name of your S3 bucket for uploaded documents, as shown below:
+
+```
+profile_name=<AWS_CLI_PROFILE_NAME>
+save_folder=<YOUR_S3_BUCKET_NAME>
```

Comment:
You have listed an S3 Bucket is required. Can you please ensure the prerequisites include needing the bucket and having access to the bucket?